### PR TITLE
feat: playlist management

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Changes

- user can create a playlist
- user can delete playlist 
- user can add videos to playlist 
- user can remove videos from playlist

image

### CODE REVIEW

File: src/components/Modal/Modal.jsx [hunk 1]
```diff
@@ -32,12 +30,35 @@ function Modal({video ,videoId}) {
...
+    setShowModal(prev => ({...prev , playlistModal: true}))
+    setShowModal(prev => ({...prev , modal: false}))
+  }
+
+  const removeFromPlaylistHandler = async () => {
+    try{
+      const resp = await axios.delete(`/api/user/playlists/${playlistId}/${videoId}` , {
+        headers: {authorization: token}
+      })
+      console.log(resp);
```
Comment (tushargoyal812) [inline src/components/Modal/Modal.jsx:43]: remove console

File: src/pages/VideoDetails/VideoDetails.jsx [hunk 2]
```diff
@@ -1,58 +1,138 @@
...
+  const addToLiked = async (video) => {
+    try {
+      const resp = await axios.post(
+        "/api/user/likes",
+        { video },
+        {
+          headers: { authorization: token },
+        }
+      );
+      console.log(resp);
```
Comment (tushargoyal812) [inline src/pages/VideoDetails/VideoDetails.jsx:29]: also here

**Review by tushargoyal812 — COMMENTED** [review]
nothing much...good to merge

File: src/App.js [hunk 3]
```diff
@@ -7,6 +7,8 @@ import {VideoDetails} from './pages/VideoDetails/VideoDetails';
 import {Like} from './pages/Like/Like';
 import {WatchLater} from './pages/WatchLater/WatchLater'
 import {History} from './pages/History/History'
+import {Playlist} from './pages/Playlist/Playlist'
+import { PlaylistDetails } from './pages/PlaylistDetails/PlaylistDetails';
```
Comment (dusariudaykumar) [inline src/App.js:11]: Just a suggestion,  create ``index.js`` file and then export all the pages you need in your app from there

**Review by dusariudaykumar — APPROVED** [review]
- The UI looks great 
- The login with test credentials is not working other than that very thing looks fine. Good work!

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

File: src/pages/VideoDetails/VideoDetails.jsx (hunk 2 above)
Comment (sharmayatendra) [reply src/pages/VideoDetails/VideoDetails.jsx:29]: ok

File: src/App.js (hunk 3 above)
Comment (sharmayatendra) [reply src/App.js:11]: i did that may be i forgot here

Comment (sharmayatendra) [reply src/App.js:11]: login with test is also working fine

**Pull request merged**